### DISCUSSION TITLE
Actividades de Frontend

### DISCUSSION THREAD

**Discussion by OscarJG20** [Launch-X-Latam/MisionFrontEnd#420] [Show and tell]
Hola!
No sé si aún haya gente activa, pero por aquí adjunto mis actividades. Cualquier duda, comentario o sugerencia se los agradeceré :"D

[Proyectos: Launch-X-Frontend](https://github.com/OscarJG20/Launch-X-Frontend)

**Comment by GonDev1** [comment]
> Hola!
> No sé si aún haya gente activa, pero por aquí adjunto mis actividades. Cualquier duda, comentario o sugerencia se los agradeceré :"D
> 
> [Proyectos: Launch-X-Frontend](https://github.com/OscarJG20/Launch-X-Frontend)

Revise tus prácticas y me parecen muy pro, además de .js ¿Usas algo más? Cómo las media querys, espero que se sumen más comentarios y hacer un pequeña comunidad, saludos ✌️😎

**Reply by OscarJG20** [reply]
Sí, uso un par de Media queries, pero son randoms (no son las especificadas para cada tipo de dispositivo). Y pues creo que eso es todo. En tu actividad de la página de tacos es curioso como manejaste el responsive sin usar (creo) nada de media queries, de hecho creo que eso es lo que se debe realizar, aunque hay casos donde sí que son necesarias las medias. Ánimo y continúa esforzándote :"D

**Reply by GonDev1** [reply]
Gracias Oscar

Como lo mencionas, no use media querys para ello pero si que hacen falta para que las cajas se acomoden mejor, aunque ya estoy estudiando el tema, gracias por darte una vuelta por mi repo, saludos 😉
